features/project_version.feature

```diff
@@ -0,0 +1,130 @@
+@api @test
+Feature: Project version
+
+  Users should be able to retrieve project versions.
+
+  The details of a test contains:
+  - name
+  - project id
+  - creation date
+
+
+
+  Background:
+    Given private organization Rebel Alliance exists
+    And user hsolo who is a member of Rebel Alliance exists
+    And project X-Wing exists within organization Rebel Alliance
+    And project version 1.0.1 exists for project X-Wing with creation date set 2 days ago
```

Comment:
`with creation date set 2 days ago` isn't very human friendly. I would prefer something like "project version 1.0.1 exists for project X since 2 days ago"